### ISSUE TITLE
Upload on checkout not saving

### ISSUE THREAD

**Issue by brashrebel** [easydigitaldownloads/edd-upload-file#10]
Customer reported and I just replicated. When a customer uploads a file on the checkout page, it goes nowhere. Payment completes but no `edd_upload_file_files` value is created for the payment record. On the receipt page, however, it works fine.

**Comment by pippinsplugins** [comment]
@ghost1227 could we look at this please?

**Comment by ghost1227** [comment]
@brashrebel @pippinsplugins I've tested this every way I can think of, and it works for me every time. Can you give me more details as to how your site is setup? What settings you have for the plugin? Anything?

![screen shot 2016-06-30 at 5 55 29 am](https://cloud.githubusercontent.com/assets/36433/16485978/70eae7dc-3e87-11e6-9a57-3f76f62d9852.PNG)
![screen shot 2016-06-30 at 5 55 53 am](https://cloud.githubusercontent.com/assets/36433/16485977/70c73d14-3e87-11e6-84ba-f74ed6db3313.PNG)

**Comment by brashrebel** [comment]
Got started testing but ran out of time. I tried with different themes, different users, fewer plugins, etc. I have never successfully gotten the upload on checkout to save. Lots of attempts, zero successes. This is on a local site with not much besides a few EDD extensions.

Just playing around I used:

```
add_action( 'template_redirect', function() {
    var_dump(EDD()->session);
} );
```
and `edd_upload_files` is never set at any point that I can see.

**Comment by brashrebel** [comment]
Myself and the customer are still experiencing this issue and they are now asking for a refund. Is this likely to be addressed in the near future?

**Comment by ghost1227** [comment]
@brashrebel I still can't replicate. Can't fix a problem that (as far as I can tell) doesn't exist...

**Comment by ghost1227** [comment]
Can you get me SOME sort of idea as to how to actually replicate??

**Comment by michaelbeil** [comment]
This seems to be fixed or can be closed as of 2.0.0.
